linera-service/src/linera.rs

```diff
@@ -579,6 +582,21 @@ struct ClientOptions {
     wait_for_outgoing_messages: bool,
 }
 
+impl ClientOptions {
+    fn init() -> Result<Self, anyhow::Error> {
+        let mut client_options = ClientOptions::from_args();
+        let wallet_env_var = env::var("LINERA_WALLET").ok();
```

Comment:
@christos-h Have you considered https://docs.rs/structopt/latest/structopt/#environment-variable-fallback ?

Reply:
Ah, I didn't know about this. Much better! :+1: